unity_jobs_simulator/Assets/Scripts/Robot_Controller.cs

```diff
@@ -140,17 +156,16 @@ void FixedUpdate()
             setIncAngles(); // This calculates the incremental angles given the increments and the target angles
             _enroute = new List<float>(targetPos); // We set our enroute position as this is where our robot movement will be moving to
             _nextAngles = addLists(_currentDrivePos, _incAngles); // This increments the angles by one increment
-            _t1 = DateTime.Now; // Time.realtimeSinceStartup; // We record the start time for this motion
+            _t1 = _startTime.Subtract(DateTime.Now); // Time.realtimeSinceStartup; // We record the start time for this motion
```

Comment:
This change feels unrelated to this PR. Generally, separate features should be separate PRs. You can leave this here, but just something to note for future reference.